recipe/meta.yaml

```diff
@@ -163,42 +165,43 @@ outputs:
         - {{ pin_compatible('numpy') }}
         - packaging >=23.2
         - panda-client >=1.5.70
-        - pandas >=2.1.4,<3
+        - pandas >=2.1.4
```

Comment:
I would think we would want to update the minimum and keep the maximum since breakage might be expected, although we can always add it later if need be.

Reply:
Looks like 2.2.3 is the current so at least `>=2.2,<3` seems reasonable.